addons/backups-restic/backups-restic.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: s3-credentials
+  name: backup-restic-s3-credentials
```

Comment:
@kron4eg @xmudrii This or more specific `kubeone-backup-restic-credentials`?

Reply:
I was planning to go with `kubeone-backups-credentials`.

Reply:
That sounds fine as well, my intention behind not adding the `kubeone-` prefix but we already use that prefix for other credentials. So let's do the same here.

Reply:
Updated PTAL